### PR TITLE
added a utility and removed useless class

### PR DESCRIPTION
**Pull request by rochestb**

### CODE REVIEW

File: source/css/app/defaults/_grid.scss [hunk 1]
```diff
@@ -11,10 +11,4 @@
 .grid-centered {
   float: none;
   margin: 0 auto;
-}
-
-.vertical-half {
-  height: 50%;
-  overflow: hidden;
-  position: relative;
-}
```
Comment (rochestb) [inline source/css/app/defaults/_grid.scss]: Can anyone tell me what this does?

File: source/css/app/utilities/_general.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  float: left;
+}
+
+.pull-right {
+  float: right;
+}
+
+.pull-none {
+  float: none;
+}
```
Comment (rochestb) [inline source/css/app/utilities/_general.scss]: I find myself using these a lot and it's been nice to add them to my personal projects.

File: source/css/app/defaults/_grid.scss (hunk 1 above)
Comment (rochestb) [inline source/css/app/defaults/_grid.scss]: @tsvensen

File: source/css/app/utilities/_general.scss (hunk 2 above)
Comment (bmsterling) [inline source/css/app/utilities/_general.scss]: These look fine, but we should move these to _grid.scss

Comment (tsvensen) [inline source/css/app/utilities/_general.scss]: Float left is already apart of the grid system.  You can use .flow-opposite for float right.

File: source/css/app/defaults/_grid.scss (hunk 1 above)
Comment (tsvensen) [inline source/css/app/defaults/_grid.scss]: I'm not sure where that came from.

**Pull request closed without merging**